compiler/native/native.go

```diff
@@ -210,3 +211,12 @@ func (c *client) WithUser(u *library.User) compiler.Engine {
 
 	return c
 }
+
+// WithLabels sets the label(s) in the Engine.
+func (c *client) WithLabels(labels []string) compiler.Engine {
```

Comment:
Can you add a test func for this? In `native_test.go`

Reply:
Done